### PR TITLE
Fix/small fixes june

### PR DESCRIPTION
**Pull request by CindyvdVries**

### CODE REVIEW

File: components/disclaimer-modal.vue [hunk 1]
```diff
@@ -72,6 +69,23 @@
         this.open = false
       }
     },
+    mounted() {
+      fetch('/docs/user-agreements.md')
```
Comment (SiggyF) [inline components/disclaimer-modal.vue:73]: why fetch it?  So it can be changed between releases?

File: components/graph-line.vue [hunk 2]
```diff
@@ -330,12 +330,13 @@
 </script>
 
 <style>
-  .graph-line__aspect-ratio.image {
+  .graph-line__aspect-ratio--image {
     height: 1200px;
   }
 
-  .graph-image {
+  .graph-line__image {
```
Comment (SiggyF) [inline components/graph-line.vue:337]: `--` or `__`?

File: layouts/default.vue [hunk 3]
```diff
@@ -14,11 +14,9 @@
         ref="map"
         :access-token="mapboxAccessToken"
         :preserve-drawing-buffer="true"
-        @mb-load="setStyleLayers"
-        :zoom="1"
-        :center="[0, 0]"
-        map-style="mapbox://styles/global-data-viewer/ckaqyfcc63q1w1io3l3bpd50h?fresh=true"
+        map-style="mapbox://styles/global-data-viewer/cjtss3jfb05w71fmra13u4qqm?fresh=true"
```
Comment (SiggyF) [inline layouts/default.vue:17]: fresh=true is only needed for debugging

File: components/disclaimer-modal.vue (hunk 1 above)
Comment (CindyvdVries) [reply components/disclaimer-modal.vue:73]: because the file wasn't loaded properly as a string otherwise

File: components/graph-line.vue (hunk 2 above)
Comment (CindyvdVries) [reply components/graph-line.vue:337]: __ is consistent with rest of app

**Pull request merged**